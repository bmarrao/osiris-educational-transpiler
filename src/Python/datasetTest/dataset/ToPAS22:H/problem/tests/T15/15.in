5
12 17
qZWD xHhYI w SBVbE fZs Frz eKUq Z yKPG BaTCv WX o
w xHhYI
qZWD Frz
o fZs
Z xHhYI
BaTCv w
Z qZWD
SBVbE o
o Frz
WX xHhYI
Frz Z
qZWD o
yKPG BaTCv
w qZWD
SBVbE yKPG
w WX
BaTCv xHhYI
qZWD xHhYI
12 33
iC l o vHPU h tn lG qUnT PB oy PK DnLJ
qUnT PK
iC tn
PB oy
tn o
vHPU oy
vHPU iC
oy h
oy iC
PB qUnT
vHPU DnLJ
l DnLJ
iC qUnT
vHPU l
o iC
DnLJ PB
tn DnLJ
o PK
PK iC
o oy
h vHPU
tn PB
qUnT o
PK oy
vHPU PK
h PB
DnLJ o
h l
tn PK
o PB
h PK
qUnT vHPU
l tn
tn h
12 29
DjF Hpbj Nn VL AjS JicX J BE memCv afOUX GOLEL sVyo
Nn afOUX
DjF AjS
J afOUX
memCv J
Hpbj J
BE memCv
J BE
DjF VL
AjS J
sVyo JicX
BE Hpbj
sVyo VL
memCv VL
AjS GOLEL
memCv JicX
DjF Nn
Hpbj memCv
VL afOUX
JicX J
sVyo GOLEL
VL BE
memCv sVyo
afOUX BE
Hpbj sVyo
DjF memCv
GOLEL J
Hpbj AjS
BE GOLEL
Nn AjS
12 0
EaDPH xR ybh aN it N sHJq lUqT j lS JOR ZV
12 25
Qi J muy oMblx cZ VFh ktfOf nnfT xLVXz Tbm tw MeaRo
Tbm MeaRo
J Tbm
cZ Qi
ktfOf J
cZ muy
muy MeaRo
MeaRo xLVXz
Tbm oMblx
cZ oMblx
VFh muy
cZ nnfT
oMblx ktfOf
tw ktfOf
MeaRo cZ
tw cZ
muy Qi
MeaRo ktfOf
Tbm cZ
cZ VFh
xLVXz muy
tw Qi
MeaRo Qi
cZ xLVXz
oMblx muy
muy Tbm
