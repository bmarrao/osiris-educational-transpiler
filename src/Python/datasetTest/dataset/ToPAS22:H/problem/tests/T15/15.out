Possivel: 71009280
Possivel: 5760000
Possivel: 13363200
Possivel: 479001600
Possivel: 10321920
